templates/files/conf/setup-kubelet-environment

```diff
@@ -8,15 +8,306 @@ EnvFile="/etc/kubelet-environment"
+r5dn.12xlarge)  MAX_PODS=$((234  - $reserved_ips)) ;;
+r5dn.16xlarge)  MAX_PODS=$((737  - $reserved_ips)) ;;
+r5dn.24xlarge)  MAX_PODS=$((737  - $reserved_ips)) ;;
+r5n.large)      MAX_PODS=$((29   - $reserved_ips)) ;;
+r5n.xlarge)     MAX_PODS=$((58   - $reserved_ips)) ;;
+r5n.2xlarge)    MAX_PODS=$((58   - $reserved_ips)) ;;
+r5n.4xlarge)    MAX_PODS=$((234  - $reserved_ips)) ;;
+r5n.8xlarge)    MAX_PODS=$((234  - $reserved_ips)) ;;
+r5n.12xlarge)   MAX_PODS=$((234  - $reserved_ips)) ;;
+r5n.16xlarge)   MAX_PODS=$((737  - $reserved_ips)) ;;
+r5n.24xlarge)   MAX_PODS=$((737  - $reserved_ips)) ;;
+t1.micro)       MAX_PODS=$((4    - $reserved_ips)) ;;
+t2.nano)        MAX_PODS=$((4    - $reserved_ips)) ;;
+t2.micro)       MAX_PODS=$((4    - $reserved_ips)) ;;
+t2.small)       MAX_PODS=$((11   - $reserved_ips)) ;;
+t2.medium)      MAX_PODS=$((17   - $reserved_ips)) ;;
+t2.large)       MAX_PODS=$((35   - $reserved_ips)) ;;
+t2.xlarge)      MAX_PODS=$((44   - $reserved_ips)) ;;
+t2.2xlarge)     MAX_PODS=$((44   - $reserved_ips)) ;;
+t3.nano)        MAX_PODS=$((4    - $reserved_ips)) ;;
+t3.micro)       MAX_PODS=$((4    - $reserved_ips)) ;;
+t3.small)       MAX_PODS=$((11   - $reserved_ips)) ;;
+t3.medium)      MAX_PODS=$((17   - $reserved_ips)) ;;
+t3.large)       MAX_PODS=$((35   - $reserved_ips)) ;;
+t3.xlarge)      MAX_PODS=$((58   - $reserved_ips)) ;;
+t3.2xlarge)     MAX_PODS=$((58   - $reserved_ips)) ;;
+t3a.nano)       MAX_PODS=$((4    - $reserved_ips)) ;;
+t3a.micro)      MAX_PODS=$((4    - $reserved_ips)) ;;
+t3a.small)      MAX_PODS=$((8    - $reserved_ips)) ;;
+t3a.medium)     MAX_PODS=$((17   - $reserved_ips)) ;;
+t3a.large)      MAX_PODS=$((35   - $reserved_ips)) ;;
+t3a.xlarge)     MAX_PODS=$((58   - $reserved_ips)) ;;
+t3a.2xlarge)    MAX_PODS=$((58   - $reserved_ips)) ;;
+u-6tb1.metal)   MAX_PODS=$((147  - $reserved_ips)) ;;
+u-9tb1.metal)   MAX_PODS=$((147  - $reserved_ips)) ;;
+u-12tb1.metal)  MAX_PODS=$((147  - $reserved_ips)) ;;
+x1.16xlarge)    MAX_PODS=$((234  - $reserved_ips)) ;;
+x1.32xlarge)    MAX_PODS=$((234  - $reserved_ips)) ;;
+x1e.xlarge)     MAX_PODS=$((29   - $reserved_ips)) ;;
+x1e.2xlarge)    MAX_PODS=$((58   - $reserved_ips)) ;;
+x1e.4xlarge)    MAX_PODS=$((58   - $reserved_ips)) ;;
+x1e.8xlarge)    MAX_PODS=$((58   - $reserved_ips)) ;;
+x1e.16xlarge)   MAX_PODS=$((234  - $reserved_ips)) ;;
+x1e.32xlarge)   MAX_PODS=$((234  - $reserved_ips)) ;;
+z1d.large)      MAX_PODS=$((29   - $reserved_ips)) ;;
+z1d.xlarge)     MAX_PODS=$((58   - $reserved_ips)) ;;
+z1d.2xlarge)    MAX_PODS=$((58   - $reserved_ips)) ;;
+z1d.3xlarge)    MAX_PODS=$((234  - $reserved_ips)) ;;
+z1d.6xlarge)    MAX_PODS=$((234  - $reserved_ips)) ;;
+z1d.12xlarge)   MAX_PODS=$((737  - $reserved_ips)) ;;
+z1d.metal)      MAX_PODS=$((737  - $reserved_ips)) ;;
+*)              MAX_PODS=40 ;;
+esac
+
+{{ else }}
 MAX_PODS=110
-{{end}}
+{{ end }}
 
-echo "MAX_PODS=${MAX_PODS}" >> ${EnvFile}
+echo "MAX_PODS=${MAX_PODS}" >> ${env_file}
```

Comment:
@calvix because this thing alwas appends we'll have a new line after each restart. Like:

```
WARNING: CONTROL-PLANE MASTER | giantswarm@ip-10-0-5-45 ~ $ cat /etc/kubelet-environment
MAX_PODS=24
MAX_PODS=24
```

Should the file be removed just before this line? Like:

```suggestion
rm -f ${env_file}
echo "MAX_PODS=${MAX_PODS}" >> ${env_file}
```